1.0.0/reference.json

```diff
@@ -151,8 +151,8 @@
         "default-meaning": "Add the current symbolizer on top of other symbolizer.",
         "doc": "Composite operation. This defines how this symbolizer should behave relative to symbolizers atop or below it.",
         "type": [
-          "add",
-          "overlay"
+            "plus",
+            "src-over"
```

Comment:
what about multiply?

Reply:
It has not been added since Tangram doesn't support it completely (unsupported if features aren't opaque). `plus` and `src-over` are in because support is a little bit better and because I'm working with them to fix them, fixing `multiply` in Tangram is trickier.